src/Components/v2/AuctionFAQ.tsx

```diff
@@ -0,0 +1,186 @@
+import { Box, media, Serif, space, Spacer, Theme, Toggle } from "@artsy/palette"
+import React from "react"
+import styled from "styled-components"
+
+export const AuctionFAQ: React.FC = () => {
+  return (
+    <Theme>
+      <AuctionWrapper>
+        <Serif size="8" color="black100">
+          Auction FAQs
+        </Serif>
+        <Spacer mb={2} />
+        <Serif size="3" color="black100">
+          How can we help you? Below are answers to some of the most common
```

Comment:
Currently, we serve this content via the `Page` model in gravity which is easily edited by admins and also powers the other `multi_page` instances.

Given we also want to follow up with the collector FAQ which follows a similar pattern, it would probably make sense to start building it out that way!

If you're worried about time/blocking a release then we can merge this and follow up when we do the collector FAQ (but perhaps add a `TODO` somewhere in here).

Reply:
Made this update, now it's being served from metaphysics